HouseRules_Essentials/Rulesets/QuickandDeadRuleset.cs

```diff
@@ -0,0 +1,21 @@
+namespace HouseRules.Essentials.Rulesets
+{
+    using HouseRules.Essentials.Rules;
+    using HouseRules.Types;
+
+    internal static class QuickandDeadRuleset
+    {
+        internal static Ruleset Create()
+        {
+            const string name = "Quick and the Dead";
+            const string description = "A Skirmish mode with a small hand but fast turn around time on cards means you need to not hesitate.";
+
+            return Ruleset.NewInstance(
+                name,
+                description,
+                new CardEnergyFromAttackMultipliedRule(1.5f),
+                new CardEnergyFromRecyclingMultipliedRule(1.5f),
+                new CardLimitModifiedRule(7);
```

Comment:
I believe this is missing an extra `)` right before the semicolon.

Reply:
Yes! Corrected.

Perhaps sections for Skirmish and Multiplayer rules in a Stack lay out would be better but then wouldn't we need a way to tag a ruleset to fall under each 'group' ?

Reply:
When a ruleset is created, all of its rules are checked.  If they're all multiplyaer safe, then the ruleset is multiplayer safe as well.  If not, then the ruleset is marked as not safe for multiplayer.  https://github.com/orendain/DemeoMods/blob/main/HouseRules_Core/Types/Ruleset.cs#L49

So there is indeed now a way to check if a ruleset is safe! 👍🏾